.github/ISSUE_TEMPLATE/hackathon-project-template.md

```diff
@@ -119,7 +120,18 @@ or use one of the tables below
 
 | software   | level of expertise required   |
 |------------|-------------------------------|
-| ???        | beginner / confirmed / expert |
+| Brainstorm | beginner / confirmed / expert |
+| other      | beginner / confirmed / expert |
+
+**Data standards**
+
+| Standard   | level of expertise required   |
+|------------|-------------------------------|
+| BIDS       | beginner / confirmed / expert |
+| NWB        | beginner / confirmed / expert |
```

Comment:
What is NWB? :laughing:

Reply:
neurodata without borders - it's a standard that's been introduced over the past few years for neurophys data, etc

Reply:
Learning new things every day.